src/components/Slider/SliderItem/SliderItem.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { calculateDiscount } from '../../../utlis/utlis'
+
+function SliderItem({ price, oldPrice, imageSrc }) {
+    return (
+        <div className="card flex-grow-0 flex-shrink-0">
+            <img src={imageSrc} alt="" class="card-img-top" />
```

Comment:
Empty alts are going to throw warnings in the console and are not the best a11y-wise

Reply:
fix planned during json data file maping implementation